test/foundry-tests/script/DeployRelayOrderReactor.t.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import {Test} from "forge-std/Test.sol";
+import {SignedOrder} from "UniswapX/src/base/ReactorStructs.sol";
+import {DeployPermit2} from "UniswapX/test/util/DeployPermit2.sol";
+import {DeployRelayOrderReactor} from "../../../script/DeployRelayOrderReactor.s.sol";
+import {RelayOrder, OrderInfo, Input, FeeEscalator} from "../../../src/base/ReactorStructs.sol";
+import {RelayOrderReactor} from "../../../src/reactors/RelayOrderReactor.sol";
+import {RelayOrderLib} from "../../../src/lib/RelayOrderLib.sol";
+import {MockERC20} from "../util/mock/MockERC20.sol";
+import {MockUniversalRouter} from "../util/mock/MockUniversalRouter.sol";
+import {OrderInfoBuilder} from "../util/OrderInfoBuilder.sol";
+import {InputBuilder} from "../util/InputBuilder.sol";
+import {FeeEscalatorBuilder} from "../util/FeeEscalatorBuilder.sol";
+import {RelayOrderBuilder} from "../util/RelayOrderBuilder.sol";
+import {PermitSignature} from "../util/PermitSignature.sol";
+
+contract DeployRelayOrderReactorTest is Test, PermitSignature, DeployPermit2 {
+    using RelayOrderLib for RelayOrder;
+    using OrderInfoBuilder for OrderInfo;
+    using InputBuilder for Input;
+    using FeeEscalatorBuilder for FeeEscalator;
+    using RelayOrderBuilder for RelayOrder;
+
+    DeployRelayOrderReactor deployer;
+    MockERC20 tokenIn;
+    MockERC20 tokenOut;
+    uint256 constant ONE = 10 ** 18;
+
+    event Fill(bytes32 indexed orderHash, address indexed filler, address indexed swapper, uint256 nonce);
+
+    function setUp() public {
+        deployPermit2();
+        deployer = new DeployRelayOrderReactor();
+        tokenIn = new MockERC20{salt: 0x00}("Token A", "TA", 18);
+        tokenOut = new MockERC20{salt: 0x00}("Token B", "TB", 18);
+    }
+
+    function testDeploy() public {
+        address mockUniversalRouter = address(new MockUniversalRouter());
+        RelayOrderReactor reactor = deployer.run(mockUniversalRouter);
+
+        assertEq(address(reactor.PERMIT2()), 0x000000000022D473030F116dDEE9F6B43aC78BA3);
```

Comment:
would just assert that the universalRouter address within the contract is the passed in variable